### ISSUE TITLE
Soundcloud embeds are shown too small in viewer

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#737]
This was originally found in tests that @jwirfs-brock documented with a [screen capture](http://www.youtube.com/watch?v=I1-iwLeDW1Q&feature=youtu.be). Issue appears at around 8:10.  

![screenshot from 2013-04-30 11 52 08](https://f.cloud.github.com/assets/110420/445197/5ea2c358-b1b6-11e2-80df-37967de0913e.png)
## Suspected cause

There's JavaScript that auto-sizes the container elements (with borders) based on the size of the inside content.  I'm thinking that perhaps this is called before the content of the Soundcloud embed iframe is loaded.  Therefore, it doesn't have size.

**Comment by ghing** [comment]
I tried to replicate this and found that I couldn't.  This is what I tried:
1. Create a new story using a sandbox template
2. Edit the title of the story to get it to auto-save
3. Select "Untitled section" from the table of contents
4. Change the layout to side-by-side
5. In the left container, click the "Audio" icon to add an audio asset
6. In the "Enter audio url" field, paste a Soundcloud URL (such as https://soundcloud.com/djflipside33/dj-flipside-chicago-house-2010)
7. Click "Save Changes"
8. Click the "Preview" link in the upper-right-hand corner of the screen

I think it might be some kind of bandwidth-related race condition that's dependent on the order of requests of the embedded content.  I need to think about this a bit more to figure out a good way to replicate this (and similar cases)
